src/firecracker/build.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::process::Command;
+
+// this build script is called on every firecracker package build,
+// embedding the FIRECRACKER_VERSION directly in the resulting binary
+fn main() {
+    let git_version_command = Command::new("git").args(&["describe", "--dirty"]).output();
+
+    match git_version_command {
+        Ok(result) => println!(
+            "cargo:rustc-env=FIRECRACKER_VERSION={}",
+            remove_leading_v(String::from_utf8(result.stdout).unwrap())
```

Comment:
Can we do `result.stdout[1..]` instead of `remove_leading_v` ?

Reply:
Or `result.trim_start_matches`

Reply:
the `remove_leading_v` function takes into account the UTF-8 encoding. By simply doing `result.stdout[1..]`, we assume that the command returns ASCII characters.

However, the `trim_start_matches` looks perfect. I wasn't aware of this method

Reply:
done